ClientApp/src/Pages/Conservation/ConservationData.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Nice having this as its own file.
I _might_ suggest not bothering with json though and just making it a normal `.ts` file.

That way:
- our data can have types! :) 
- we can have comments and stuff if we want to
- we don't have to bother importing json files etc (although I thing Create React App might handle this for us)

ie something like

```ts
interface ConservationCardData {
    imageSrc: string;
    text: string;
    siteLink: string;
}

export const ConservationData: ConservationCardData[] = [
    { /* blah */ },
    { /* blah */ },
    { /* blah */ },
]
```

Reply:
json file is now in typescript